ext/span.c

```diff
@@ -140,6 +140,11 @@ void ddtrace_open_span(ddtrace_span_data *span) {
     // All open spans hold a ref to their stack
     ZVAL_OBJ_COPY(&span->property_stack, &stack->std);
 
+    span->duration_start = _get_nanoseconds(USE_MONOTONIC_CLOCK);
+    // Start time is nanoseconds from unix epoch
+    // @see https://docs.datadoghq.com/api/?lang=python#send-traces
+    span->start = _get_nanoseconds(USE_REALTIME_CLOCK);
```

Comment:
This seems unrelated to the other changes. Are we expecting these to be set elsewhere, and this is being defensive, or was this a bug?

Reply:
I see it's related, but I need to read the docs.

Reply:
Yeah, using the start time afterwards for the high bits.